src/text.c

```diff
@@ -161,6 +162,15 @@ main(int argc, char *argv[])
 				}
 			}
 			break;
+		case 'R':
+			conf->use_netrc = 1;
+			if (optarg) {
+				if (!sscanf(optarg, "%s", conf->netrc_filename)) {
```

Comment:
why `sscanf`?

Reply:
`sscanf` was used to keep consistency with surrounding code.

Reply:
`strlcpy` is better.

Reply:
I was afraid of using strlcpy because it is not standard (neither ISO C nor POSIX), and thus could cause portability problems. But if it is ok for you, I'll use it.

Reply:
PS: I couldn't use `strlcpy` under Linux, it is not available in libc. I'm afraid that to use that we'd have to implement it or include another lib on the project. I can use the approach I used elsewhere to emulate `strlcpy`: by using `strncpy` and null-terminating it.